BUILD

```diff
@@ -5,12 +5,20 @@
 #  bazel test ...
 
 load("@com_github_google_rules_install//installer:def.bzl", "installer")
+load("@rules_license//rules:license.bzl", "license")
 
-licenses(["notice"])  # Apache 2.0
```

Comment:
Ummm.  I was wrong about this.  I checked but the google compliance linter is going to complain about it missing.